dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateResource.java

```diff
@@ -802,4 +805,52 @@ public final Response delete(@Context final HttpServletRequest  request,
                 new BulkResultView(deletedTemplatesCount,0L,failedToDelete)))
                 .build();
     }
+
+    /**
+     * Return live version {@link com.dotmarketing.portlets.templates.model.Template} based on the id
```

Comment:
Javadoc doesn't seem to match the method's definition, or it's name is not right.